doc/manuals/admin/cli.md

```diff
@@ -121,4 +121,19 @@ The list of available options is the following:
     Active-Active Orion configuration "none" is recommended.
 -   **-mutexTimeStat**. Include semaphore waiting time in the
     "/statistics" information. It may have performance impact.
--   **-subCacheIval**. Interval in seconds between calls to Subscription Cache refresh.
+-   **-subCacheIval**. Interval in seconds between calls to subscription cache refresh. Zero
+    value means "no refresh" (default value and the recomnended one for mono-CB
+    deployments).
+-   **-noCache**. Disables context subscription cache, so subscriptions searchs are
+    always done at DB (not recommneded but useful for debugging).
+-   **-notificationMode** *(Experimental option)*. Allows to select notification mode, either:
```

Comment:
I foresee potential changes in the syntax of this CLI option (e.g. to specify pipeling, pool size, etc.) so better to mark it as experimental by the moment in the documentation.

NTC (just informative).